companies/arcteryx.md

```diff
@@ -0,0 +1,25 @@
+---
+fortune_rank: number
```

Comment:
```suggestion
```

I don't think Arc'teryx is in the [Fortune500](https://fortune.com/fortune500/2020/search/) rankings, so you can take this line out!  And remove the 'fortune500' tag.  I will be more clear about this in the instructions :)